packages/core/package.json

```diff
@@ -105,7 +105,7 @@
     "npm-run-all": "4.1.5",
     "patch-package": "6.4.7",
     "playwright": "1.11.1",
-    "purgecss": "4.0.3",
+    "purgecss": "4.1.3",
```

Comment:
These were bumped to take advantage of small improvements in the CSS minification/optimization builds.